scripts/js/index.js

```diff
@@ -292,10 +294,10 @@ function updateTopClientsTable(blocked) {
     let url, percentage;
     const sum = blocked ? data.blocked_queries : data.total_queries;
 
-    // Add note if there are no results (e.g. privacy mode enabled)
+    // Remove table if there are no results (e.g. new
```

Comment:
> e.g new installation...

This is an important distinction here.

Can we not just check the API to see what privacy mode we're on and display/hide accordingly?

Reply:
Good point. I addressed this in the second commit.